digitalpy/core/parsing/impl/json_format.py

```diff
@@ -1,4 +1,4 @@
-from digitalpy.parsing.impl.hierarchical_format import HirearchicalFormat
+from digitalpy.core.parsing.impl.hierarchical_format import HirearchicalFormat
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>D100 Missing docstring in public module